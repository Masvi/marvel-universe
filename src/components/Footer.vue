<template>
  <div class="footer">
    <div class="footer__info">
      <slot />
    </div>  
  </div>
</template>

<script>
export default {
  name: 'Footer'
}
</script>

<style lang="scss" scoped>

.footer {
  display: flex;
  justify-content:center;
  align-items:center;

  font-weight: 600;
  background-color: $secondary-red;
  height: 2.5rem;
  left: 0;
  bottom: 0;
  width: 100%;
  margin-top: 1rem;
  position: fixed;

  &__info {
    color: $primary-red;
    font-size: .75rem;
  }
}
</style>