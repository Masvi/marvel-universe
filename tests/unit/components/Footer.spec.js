import { shallowMount } from '@vue/test-utils'
import Footer from '@/components/Footer.vue';

const wrapper = shallowMount(Footer)

describe('Footer.vue', () => {
  it('Footer should be a valid vue component', () => {
    expect(wrapper).toMatchSnapshot();
  })
});
