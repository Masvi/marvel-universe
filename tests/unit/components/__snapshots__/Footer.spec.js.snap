// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue Footer should be a valid vue component 1`] = `
<div class="footer">
  <div class="footer__info"></div>
</div>
`;
